.ci/variables.json

```diff
@@ -1,4 +1,9 @@
 {
+  "prerequisite_versions": {
+    "PIP": "21.2.4",
+    "SETUPTOOLS": "58.0.4",
+    "WHEEL": "0.37.0"
```

Comment:
I realize you're being consistent with `PY38` and `PY39` below, but what are your thoughts on using lowercase in that case? `PIP` seems weird, but maybe that's just me.

Reply:
I'm ambivalent.  They look more like constants to me when they're upper-cased